gtsfm/view_graph_estimator/cycle_consistent_rotation_estimator.py

```diff
@@ -138,7 +140,14 @@ def run(
                 valid_edges, cycle_errors, max_gt_error_in_cycle, per_edge_aggregate_error, two_view_reports, output_dir
             )
 
-        logger.info("Found %d consistent rel. rotations from %d original edges.", len(valid_edges), len(input_edges))
+        end_time = time.time()
+        duration_sec = end_time - start_time
+        logger.info(
+            "Found %d consistent rel. rotations from %d original edges in %.2f sec.",
+            len(valid_edges),
+            len(input_edges),
+            duration_sec,
+        )
```

Comment:
could you add a metric? this will drown in all the noise.

Reply:
I think we may not need one, but we could add in a follow-up PR. Logged stats indicate this part is <1 sec for all runs.

Reply:
(I believe the logged stat info is sufficient for now)

Reply:
interesting, thanks!